packages/graphback-cli/src/config/FolderConfig.ts

```diff
@@ -27,4 +25,8 @@ export interface FolderConfig {
      */
     client: string
 
-}
\ No newline at end of file
+    /**
+     * Location to save migrations
+     */
+    migrations: string
```

Comment:
So our idea is that people will push that to the repository?
I think that this will go away when we use database driven metadata right?
Just checking - no action

Reply:
Yep, that was initial idea.